tests/foreman/cli/test_import.py

```diff
@@ -78,52 +80,64 @@ def update_csv_values(csv_file, new_data=None, dirname=None):
     :param csv_file: A string. The path to a CSV file that resides
         on a remote server.
     :param new_data: A list of dictionary objects. Each object has to
-        contain an org_id key corresponding to organization_id of the record
-        being updated and the key-value pairs of attributes to be changed for
-        the particular records. For example::
+        contain an key_id key corresponding to org[anization]_id of the
+        record being updated and the key-value pairs of attributes to be
+        replaced by the particular records. For example::
 
             [
                 {
-                    u'org_id': u'1',
+                    u'key_id': u'1',
                     u'organization': u'updated_organization_name_1',
                     u'username': u'updated_user_name_1',
                 },
                 {
-                    u'org_id': u'2',
+                    u'key_id': u'2',
                     u'organization': u'updated_organization_name_2',
                     u'username': u'updated_user_name_2',
                 }
             ]
 
-    :returns: A string. The path to a locally created CSV file
+    :returns: A string. The path to a remotely created CSV file
 
     """
+    updated = False
     # if new_data is not specified, no change happens
     if new_data is None:
         return csv_file
-
     result = csv_to_dataset([csv_file])
     for change in new_data:
+        if change.get('key_id') is None:
+            continue
```

Comment:
Depends on things that we need here: skip field that doesn't have key_id or raise an exception.